Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16 AS builder
+WORKDIR /go/src/github.com/forbole/big-dipper
+COPY . ./
+RUN CGO_ENABLED=0 go build -o /build/bdjuno ./cmd/bdjuno
+
+FROM alpine:latest
+WORKDIR /root/
+COPY --from=0 /build/bdjuno ./
```

I would suggest using named containers instead of int indexes here. Also, we can simply move the `bdjuno` binary directly into the `/usr/bin` folder to make it immediately executable and known to the system as a whole: 

```suggestion
COPY --from=builder /go/src/github.com/forbole/bdjuno/build/bdjuno /usr/bin/bdjuno
```